services/payments/src/simcore_service_payments/services/payments.py

```diff
@@ -59,17 +66,30 @@ async def init_one_time_payment(
     user_id: UserID,
     user_name: str,
     user_email: EmailStr,
+    stripe_price_id: StripePriceID,
+    stripe_tax_rate_id: StripeTaxRateID,
     comment: str | None = None,
 ) -> WalletPaymentInitiated:
     initiated_at = arrow.utcnow().datetime
 
+    # NOTE: PC: please modify with your "collecting of user address" PR https://github.com/ITISFoundation/osparc-simcore/issues/5138
+    user_address = UserAddress(country="CH")
```

Comment:
I wonder if it would be better to get my PR first and adapt these changes.

Reply:
Yes no problem from me. It is just about timing. In a way, this PR is written independently of yours (just the user address is hardcoded for Switzerland). I would say let's see when App Motion prepares their side and then we can decide.